test.py

```
@@ -0,0 +1,5 @@
+print("Hello World')
```

Pylint (3.3.7)

[error] Parsing failed: 'unterminated string literal (detected at line 1) (test, line 1)'

(E0001)